src/store/selectors/config.js

```diff
@@ -5,3 +5,7 @@ export const getConfig = state => state.config;
 
 export const getCoinSymbol = state =>
   createSelector(getConfig, configData => configData.chain.symbol);
+
+export const getBuyerProxyPort = state => state.config.chain.buyerProxyPort;
+
+export const getSellerProxyPort = state => state.config.chain.sellerProxyPort;
```

Comment:
use `createSelector` with `getConfig` as above
```suggestion
export const getBuyerProxyPort = state => createSelector(getConfig, configData => configData.chain. buyerProxyPort);

export const getSellerProxyPort = state => createSelector(getConfig, configData => configData.chain. sellerProxyPort);
```